README.md

```diff
@@ -28,12 +28,17 @@ Command synopsis:
     -p=9125: Port to listen on
     -port=9125: Port to listen on
     -prefix="statsrelay": The prefix to use with self generated stats
-
+    -metrics-prefix="production.us-east-1": The prefix to use with metrics passed through statsrelay
+    -bufsize="32768": Read buffer size
+    -packetlen="1400": Max packet length. Must be lower than MTU plus IPv4 and UDP headers to avoid fragmentation.
+    -sendproto="TCP": IP Protocol for sending data - TCP or UDP
```

Comment:
`UDP` is the default here and we should lists the defaults in the docs.  Same with `-tcptimeout` being 1 second, and verbose defaults to false.